xml/installation-installation-configure_lbaas.xml

```diff
@@ -724,337 +615,403 @@ rtt min/avg/max/mdev = 3.473/3.473/3.473/0.000 ms</screen>
+| timeout_member_data       | 50000                                |
+| timeout_tcp_inspect       | 0                                    |
+| updated_at                | 2019-10-29T16:40:01                  |
++---------------------------+--------------------------------------+</screen>
    </step>
   </procedure>
  </section>
  <section>
   <title>Create Floating IPs for Load Balancer</title>
   <para>
-   To create the floating IP's for the load balancer, you will need to list the
-   current ports to get the load balancer id. Once you have the id, you can
-   then create the floating IP.
+   To create the floating IP's for the load balancer, you will need to list the ports for 
+   the load balancer. Once you have the port ID, you can then create the floating IP. 
+   Notice that the name has changed between the neutron lbass-loadbalancer and openstack 
+   loadbalancer CLI. In this case, be sure to pick the port associated with the name 
+   <literal>octavia-lb-</literal> or use the port ID that was saved when the load balancer was created.
   </para>
   <procedure>
    <step>
     <para>
-     List the current ports.
-    </para>
-<screen><?dbsuse-fo font-size="0.65em"?>&prompt.ardana;openstack port list
-+--------------+---------------+-------------------+-----------------------------------------+
-| id           | name          | mac_address       | fixed_ips                               |
-+--------------+---------------+-------------------+-----------------------------------------+
-...
-| 7e5e...6450e |               | fa:16:3e:66:fd:2e | {"subnet_id": "6fc2572c-                |
-|              |               |                   | 53b3-41d0-ab63-342d9515f514",           |
-|              |               |                   | "ip_address": "10.247.94.132"}          |
-| a3d0...55efe |               | fa:16:3e:91:a2:5b | {"subnet_id": "f00299f8-3403-45ae-ac4b- |
-|              |               |                   | 58af41d57bdc", "ip_address":            |
-|              |               |                   | "10.247.94.142"}                        |
-| ca95...36854 |               | fa:16:3e:e0:37:c4 | {"subnet_id": "6fc2572c-                |
-|              |               |                   | 53b3-41d0-ab63-342d9515f514",           |
-|              |               |                   | "ip_address": "10.247.94.133"}          |
-| da28...c3c51 | loadbalancer- | fa:16:3e:1d:a2:1c | {"subnet_id": "6fc2572c-                |
-|              | 3d917...aae53 |                   | 53b3-41d0-ab63-342d9515f514",           |
-|              |               |                   | "ip_address": "10.247.94.134"}          |
-+--------------+---------------+-------------------+-----------------------------------------+</screen>
+     Search for the load balancers port, it will be used when the floating IP is created.
+     Alternately use the bash variable that contains the loadbalancer port from the previous section.
```

Comment:
Should the `loadbalancer` in this section be load balancer?

Reply:
yes, I missed this one when breaking up loadbalancer into load balancer  outside of commandlines